testData/hours_response.json

```diff
@@ -0,0 +1,150 @@
+[
+  {
+    "id": "recmmC9oshIcu35c3",
+    "fields": {
+      "open_weekday": "Monday",
+      "open_time_digits": "1100",
+      "mutual_aid_locations": [
+        "recKl4miBt9QVF3lD",
+        "rec4gl2Z6ZMd65Zto",
+        "recjVs2Jx2XcUBW44",
+        "recKS6c1gu5KLy4y5",
+        "recMVUQd0TEGst6Gr",
+        "recD6048wWrijedPJ",
+        "rec21rsZv9U5wQPLU",
+        "recqT9r7sVzqTRE7h",
+        "reczZgGVqCkSBd2Pj",
+        "recAE9XnGl75JTmIz",
+        "recv3P2IouG0JLh0T",
+        "receftJdhnVpymofG",
+        "recn9eUzJHC3Abm8V",
+        "recUag44UYTnlWipe",
+        "reccgYd62wRSyqk1N"
+      ],
+      "close_time_digits": "1500",
```

Comment:
Random Q, but how much of these values are being input by Lizards vs Airtable generating them based on formulas?

Reply:
4 are lizards, 6 are formulas, and 2 are autogenerated links to other records in different table (the `mutual_aid_locations` ones)

Reply:
Ooooo, which 4?

Reply:
The 4 lizard ones are open_weekday, close_weekday, open_time_digits, close_time_digits